src/sass/_schemaform.scss

```diff
@@ -0,0 +1,61 @@
+// Style for schemaform
```

Comment:
Thanks for moving this. Was going to make a tech debt issue for it.

I think it makes sense to move it into modules and follow the naming convention for those files.